packages/react-wildcat/test/utils/webpackBundleValidationSpec.js

```diff
@@ -4,207 +4,156 @@ const sinon = require("sinon");
 const sinonChai = require("sinon-chai");
```

Comment:
Something funky happened here? You replaced all of the tests with unrelated tests.